drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -2109,7 +2112,7 @@ function DragAndDropBlock(runtime, element, configuration) {
             disable_go_to_beginning_button: !canGoToBeginning(),
             show_go_to_beginning_button: state.go_to_beginning_button_visible,
             screen_reader_messages: (state.screen_reader_messages || ''),
-            fixed_sizing_feedback_display: (state.fixed_sizing_feedback_display || 'none'),
+            show_fixed_sizing_feedback: (state.show_fixed_sizing_feedback || false),
```

Comment:
This can just be ``show_fixed_sizing_feedback: state.show_fixed_sizing_feedback,``.